### PR TITLE
go.mod: update etcd to fix e2e tests

### LINKED ISSUE #91937: kubelet hugepages and benchmark tests panic
**Which jobs are failing**:

- ci-kubernetes-node-kubelet-serial-hugepages
- ci-kubernetes-node-kubelet-benchmark

**Which test(s) are failing**:

Node tests

**Since when has it been failing**:

It happens occasionally and started quite long time ago.

**Testgrid link**:

https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-serial-hugepages/1270133163225518083

**Reason for failure**:

panic in protobuf code:
```
panic: invalid Go type int for field k8s_io.kubernetes.vendor.go_etcd_io.etcd.etcdserver.etcdserverpb.loggablePutRequest.value_size

goroutine 228 [running]:
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.newSingularConverter(0x6155840, 0x4590640, 0x6155720, 0xc0070af5e0, 0x60c9601, 0xc006a09c20)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/convert.go:143 +0xff5
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.NewConverter(0x6155840, 0x4590640, 0x6155720, 0xc0070af5e0, 0x203000, 0xc0087159e0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/convert.go:60 +0xd9
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.fieldInfoForScalar(0x6155720, 0xc0070af5e0, 0x451e44b, 0x9, 0x0, 0x0, 0x6155840, 0x4590640, 0x451e456, 0x2e, ...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/message_reflect_field.go:234 +0x110
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.(*MessageInfo).makeKnownFieldsFunc(0xc00867d400, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xc0087158f0, 0xc008715920, 0xc008715950, 0xc008715980)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/message_reflect.go:67 +0x938
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.(*MessageInfo).makeReflectFuncs(0xc00867d400, 0x6155840, 0x4eaf4c0, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xffffffffffffffff, 0xc0087158f0, 0xc008715920, 0xc008715950, ...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/message_reflect.go:36 +0x63
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.(*MessageInfo).initOnce(0xc00867d400)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/message.go:90 +0x19c
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.(*MessageInfo).init(0xc00867d400)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/message.go:72 +0x3c
k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl.(*messageReflectWrapper).Has(0xc00bc48090, 0x6155720, 0xc0070af500, 0xc0070af500)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/google.golang.org/protobuf/internal/impl/message_reflect_gen.go:185 +0x33
k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto.(*textWriter).writeMessage(0xc0087158c0, 0x6135260, 0xc00bc48090, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto/text_encode.go:278 +0x925
k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto.(*TextMarshaler).marshal(0x8cdfd92, 0x605b880, 0xc008715830, 0x0, 0x0, 0xc00bd3be80, 0x0, 0x1)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto/text_encode.go:86 +0x18a
k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto.(*TextMarshaler).Text(...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto/text_encode.go:44
k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto.CompactTextString(...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/github.com/golang/protobuf/proto/text_encode.go:106
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb.(*loggablePutRequest).String(...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb/raft_internal_stringer.go:182
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb.(*requestOpStringer).String(0xc013744868, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb/raft_internal_stringer.go:124 +0x180
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb.(*txnRequestStringer).String(0xc013744998, 0xc0032304c0, 0x17)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb/raft_internal_stringer.go:98 +0x47e
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb.(*InternalRaftStringer).String(0xc00843d9e0, 0x57570aaac, 0x8e6bac0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/etcdserverpb/raft_internal_stringer.go:67 +0x123
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.warnOfExpensiveGenericRequest(0xc000264960, 0xbfaf5f956a16e3eb, 0x57570aaac, 0x8e6bac0, 0x5fe2560, 0xc00843d9e0, 0x0, 0x0, 0xc00732c430, 0x7, ...)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/util.go:149 +0x487
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.warnOfExpensiveRequest(0xc000264960, 0xbfaf5f956a16e3eb, 0x57570aaac, 0x8e6bac0, 0x5fe2560, 0xc00843d9e0, 0x605b640, 0xc008714570, 0x0, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/util.go:111 +0xcc
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*applierV3backend).Apply.func1(0xc000c44440, 0xc0077787e0, 0xc0086699b0, 0xbfaf5f956a16e3eb, 0x57570aaac, 0x8e6bac0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/apply.go:118 +0x10d
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*applierV3backend).Apply(0xc000c44440, 0xc0077787e0, 0xc0086699b0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/apply.go:176 +0x2ba
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*authApplierV3).Apply(0xc0003fe050, 0xc0077787e0, 0x0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/apply_auth.go:60 +0xf5
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*EtcdServer).applyEntryNormal(0xc000c86100, 0xc013745498)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/server.go:2213 +0x1ea
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*EtcdServer).apply(0xc000c86100, 0xc00752fab8, 0x1, 0x1, 0xc0013c1ea0, 0x434dd0, 0x547f068, 0xc000080000)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/server.go:2127 +0x591
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*EtcdServer).applyEntries(0xc000c86100, 0xc0013c1ea0, 0xc0079e1100)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/server.go:1379 +0xd4
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*EtcdServer).applyAll(0xc000c86100, 0xc0013c1ea0, 0xc0079e1100)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/server.go:1103 +0x88
k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver.(*EtcdServer).run.func8(0x6081900, 0xc000e54700)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/etcdserver/server.go:1048 +0x3c
k8s.io/kubernetes/vendor/go.etcd.io/etcd/pkg/schedule.(*fifo).run(0xc000d1b2c0)
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/pkg/schedule/schedule.go:157 +0xf7
created by k8s.io/kubernetes/vendor/go.etcd.io/etcd/pkg/schedule.NewFIFOScheduler
	/go/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/vendor/go.etcd.io/etcd/pkg/schedule/schedule.go:70 +0x13b
```

log with the traceback: https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-node-kubelet-serial-hugepages/1269647959063007233/artifacts/n1-standard-2-ubuntu-gke-1804-1-16-v20200330-9717e5df/services.log

**Anything else we need to know**:

Here is a workaround I did for ci-kubernetes-node-kubelet-benchmark job:
https://github.com/kubernetes/test-infra/pull/17743#issuecomment-638107187

### LINKED ISSUE #92070: kubernetes unit tests fail with Panic messages stating "invalid type"
[logs_ci-unit-kubernetes_50_build-log.txt](https://github.com/kubernetes/kubernetes/files/4772888/logs_ci-unit-kubernetes_50_build-log.txt)

**What happened**:
Below are the **27 unit tests** that panic and FAIL while I run "make test"

Also the analysis says that the failure started occurring after the below PR merge:
#https://github.com/kubernetes/kubernetes/pull/90582/commits/283d9f7483d9a5b30746921a68e7a5e586c2c80e
This commit **updates github.com/golang/protobuf from v1.3.3 to v1.4.2.**
The invalid Type failures that occur are from the newer implementation of protobuf that is added at "kubernetes/vendor/google.golang.org/protobuf" which did not exist prior to this commit.

**Failure Message: panic: field etcdserverpb.Request.ID has invalid type: got uint64, want pointer**
1.	pkg/master TestStorageVersionHashes
2.	pkg/registry/apps/daemonset/storage TestDelete
3.	pkg/registry/apps/replicaset/storage TestList
4.	pkg/registry/core/configmap/storage TestCreate
5.	pkg/registry/core/event/storage TestDelete
6.	pkg/registry/core/persistentvolume/storage TestCreate
7.	pkg/registry/core/persistentvolumeclaim/storage TestGet
8.	pkg/registry/core/pod/storage TestEviction
9.	pkg/registry/core/service/storage
10.	pkg/registry/scheduling/priorityclass/storage TestUpdate
11.	pkg/registry/storage/csinode/storage TestCreate
12.	pkg/registry/storage/volumeattachment/storage TestWatch
13.	vendor/k8s.io/apiserver/pkg/registry/generic/registry TestDryRunCreateDoesntCreate
14.	vendor/k8s.io/apiserver/pkg/storage/etcd3 TestCreateWithTTL 
15.	vendor/k8s.io/apiserver/pkg/storage/tests TestWatchBookmarksWithCorrectResourceVers**ion
**Failure message: panic: invalid Go type int for field** k8s_io.kubernetes.vendor.go_etcd_io.etcd.etcdserver.etcdserverpb.loggablePutRequest.value_size**
16.	pkg/registry/apps/deployment/storage TestUpdate 
17.	pkg/registry/apps/statefulset/storage TestStatusUpdate 
18.	pkg/registry/batch/cronjob/storage TestCreate
19.	pkg/registry/core/namespace/storage TestCreate
20.	pkg/registry/core/node/storage TestList (
21.	pkg/registry/core/replicationcontroller/storage TestWatch
22.	pkg/registry/core/secret/storage TestCreate
23.	pkg/registry/networking/networkpolicy/storage TestCreate
24.	pkg/registry/settings/podpreset/storage TestUpdate
25.	pkg/registry/storage/csidriver/storage TestList
26.	pkg/registry/storage/storageclass/storage TestUpdate
27.	vendor/k8s.io/apiextensions-apiserver/pkg/registry/customresource

**What you expected to happen**:
The unit tests must pass without error message.

**How to reproduce it (as minimally and precisely as possible)**:
1. Setup Go lang 1.13.5
2. Clone kubernetes source code : git clone git@github.com:kubernetes/kubernetes.git
3. Ensure you are on master branch : git checkout master
4. cd kubernetes/hack/tools
5. GO111MODULE=on go install gotest.tools/gotestsum
6. make generated_files
7. make test WHAT=<Path of any one test from above failures>
         eg: make test WHAT=k8s.io/kubernetes/pkg/registry/apps/deployment/storage/

**Anything else we need to know?**:
This failure consistently occurs when unit test is run through an automated Jenkins pipeline. But the failure is seen intermittently when run "make test" manually. Sometimes, easily reproducible when we do "make clean", "make clean_generated" followed by "make test"
Please see the consistent failure at https://k8s-testgrid.appspot.com/sig-node-ppc64le#unit-tests

**Environment**:
Architechture: linux/ppc64le
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`): Ubuntu 18.04.4
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:
@mkumatag

### LINKED ISSUE #93163: pull-kubernetes-node-e2e fails half the time
**Which jobs are failing**:
pull-kubernetes-node-e2e fails about 50% of the time

**Which test(s) are failing**:

Unclear, at least some of the failures are timing out.

**Since when has it been failing**:

At least 2 days, but probably longer

**Testgrid link**:

https://prow.k8s.io/?job=pull-kubernetes-node-e2e

**Reason for failure**:

I do not know.

**Anything else we need to know**:

This is making it really hard to get any PR merged.

/sig node
/priority failing-test

### PR DESCRIPTION
**Pull request by bart0sh**
**What type of PR is this?**

/kind flake

**What this PR does / why we need it**:

This PR updates etcd to include this fix: https://github.com/etcd-io/etcd/pull/12106
It should fix increased amount of failures in e2e node tests.

**Which issue(s) this PR fixes**:

Fixes #91937
Fixes #92070
Fixes #93163

**Special notes for your reviewer**:

Delta from 3.4.9 to the latest 3.4 updates at https://github.com/etcd-io/etcd/compare/v3.4.9...release-3.4

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by liggitt** [comment]
you'll need to follow the instructions in go.mod to update the dependency:

```
// Run hack/pin-dependency.sh to change pinned dependency versions.
// Run hack/update-vendor.sh to update go.mod files and the vendor directory.
```

**Comment by bart0sh** [comment]
@liggitt thank you for the review! done.

**Comment by MHBauer** [comment]
pull-kubernetes-node-e2e successful run
/test pull-kubernetes-node-e2e

**Comment by BenTheElder** [comment]
Note that the Azure jobs are not blocking and I'm pretty sure they usually fail currently

**Comment by bart0sh** [comment]
@BenTheElder 
> Note that the Azure jobs are not blocking and I'm pretty sure they usually fail currently

Why the merging is blocked then?

**Comment by RainbowMango** [comment]
@bart0sh 
I'm curious about how you generate the version `v0.5.0-alpha.5.0.20200715164516-7b8270416d8b` used by `hack/pin-dependency.sh`. Can you share some info with me?

**Comment by bart0sh** [comment]
@RainbowMango That's done automagically by hack/pin-dependency.sh script:
```
kubernetes (master) $ hack/pin-dependency.sh go.etcd.io/etcd 7b8270416d8b
Running: go get go.etcd.io/etcd@7b8270416d8b
go: go.etcd.io/etcd 7b8270416d8b => v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
Resolved to go.etcd.io/etcd@v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
Running: go mod edit -replace go.etcd.io/etcd=go.etcd.io/etcd@v0.5.0-alpha.5.0.20200715164516-7b8270416d8b

Run hack/update-vendor.sh to rebuild the vendor directory

kubernetes (master) $ git diff | cat
diff --git a/go.mod b/go.mod
index ef03b5a1eae..16054b31dcc 100644
--- a/go.mod
+++ b/go.mod
@@ -97,7 +97,7 @@ require (
 	github.com/urfave/negroni v1.0.0 // indirect
 	github.com/vishvananda/netlink v1.1.0
 	github.com/vmware/govmomi v0.20.3
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200520232829-54ba9589114f
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
 	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
diff --git a/go.sum b/go.sum
index 74982254b0f..ebb38c9fd4a 100644
--- a/go.sum
+++ b/go.sum
@@ -469,6 +469,8 @@ go.etcd.io/bbolt v1.3.5 h1:XAzx9gjCb0Rxj7EoqcClPD1d5ZBxZJk0jbuoPHenBt0=
 go.etcd.io/bbolt v1.3.5/go.mod h1:G5EMThwa9y8QZGBClrRx5EY+Yw9kAhnjy3bSjsnlVTQ=
 go.etcd.io/etcd v0.5.0-alpha.5.0.20200520232829-54ba9589114f h1:pBCD+Z7cy5WPTq+R6MmJJvDRpn88cp7bmTypBsn91g4=
 go.etcd.io/etcd v0.5.0-alpha.5.0.20200520232829-54ba9589114f/go.mod h1:skWido08r9w6Lq/w70DO5XYIKMu4QFu1+4VsqLQuJy8=
+go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b h1:ZTyVhY1jvMHy78Lc0YdU03lKxEgpw7zvwzXYy/Tz6Dw=
+go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b/go.mod h1:skWido08r9w6Lq/w70DO5XYIKMu4QFu1+4VsqLQuJy8=
 go.mongodb.org/mongo-driver v1.1.2 h1:jxcFYjlkl8xaERsgLo+RNquI0epW6zuy/ZRQs6jnrFA=
 go.mongodb.org/mongo-driver v1.1.2/go.mod h1:u7ryQJ+DOzQmeO7zB6MHyr8jkEQvC8vH7qLUO4lqsUM=
 go.opencensus.io v0.22.2 h1:75k/FF0Q2YM8QYo07VPddOLBslDt1MZOdEslOHvmzAs=
diff --git a/staging/src/k8s.io/apiextensions-apiserver/go.mod b/staging/src/k8s.io/apiextensions-apiserver/go.mod
index cee4e0bf952..0626a38b8b1 100644
--- a/staging/src/k8s.io/apiextensions-apiserver/go.mod
+++ b/staging/src/k8s.io/apiextensions-apiserver/go.mod
@@ -18,7 +18,7 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200520232829-54ba9589114f
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
 	google.golang.org/grpc v1.27.0
 	gopkg.in/yaml.v2 v2.2.8
 	k8s.io/api v0.0.0
@@ -42,3 +42,5 @@ replace (
 	k8s.io/code-generator => ../code-generator
 	k8s.io/component-base => ../component-base
 )
+
+replace go.etcd.io/etcd => go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
diff --git a/staging/src/k8s.io/apiserver/go.mod b/staging/src/k8s.io/apiserver/go.mod
index 23864b8d532..22cadb6e45f 100644
--- a/staging/src/k8s.io/apiserver/go.mod
+++ b/staging/src/k8s.io/apiserver/go.mod
@@ -31,7 +31,7 @@ require (
 	github.com/stretchr/testify v1.4.0
 	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
 	go.etcd.io/bbolt v1.3.5 // indirect
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200520232829-54ba9589114f
+	go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
 	go.uber.org/zap v1.10.0
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
 	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
@@ -60,3 +60,5 @@ replace (
 	k8s.io/client-go => ../client-go
 	k8s.io/component-base => ../component-base
 )
+
+replace go.etcd.io/etcd => go.etcd.io/etcd v0.5.0-alpha.5.0.20200715164516-7b8270416d8b
```

**Comment by RainbowMango** [comment]
@bart0sh Got it thanks!

**Comment by jingyih** [comment]
Is this PR just testing the fix (which will be included in etcd v3.4.10), or do we intend to merge this? In the past we've only bumped etcd dependency to a released tag. e.g. https://github.com/kubernetes/kubernetes/issues/91266

**Comment by liggitt** [comment]
/hold
that's a good point... do we have an ETA on 3.4.10?

**Comment by jingyih** [comment]
> that's a good point... do we have an ETA on 3.4.10?

I would think it can be done in the next few days. I personally have not walked through the entire etcd release process, so I prefer to ask @gyuho or @jpbetz to help with the release.

**Comment by BenTheElder** [comment]
Merging was blocked due to code freeze?
Now it's blocked due to the hold.

**Comment by lavalamp** [comment]
Personally, if this is contributing to the current extremely high e2e flake rate, I'd vote for merging this now and then merging a followup later.

**Comment by lavalamp** [comment]
What is the relationship between this PR and https://github.com/kubernetes/kubernetes/pull/92349?

**Comment by gyuho** [comment]
I am doing the `v3.3.23` and `v3.4.10` release today. These are already tagged.

**Comment by MHBauer** [comment]
Looks like 18dfb9cca345bb2b2fbe73d5fc31028c2477bef1 is the hash for the [upstream tag of v3.4.10](https://github.com/etcd-io/etcd/tree/v3.4.10), so we probably need to update this PR to get the [change of version](https://github.com/etcd-io/etcd/commit/18dfb9cca345bb2b2fbe73d5fc31028c2477bef1).

**Comment by liggitt** [comment]
if we can bump this to pick up the official sha for 3.4.10, it can merge 

> What is the relationship between this PR and #92349?

This PR updates the client code and embedded server used by unit/integration tests to 3.4.10

#92349 updates the etcd image used in the gce manifests to run 3.4.9.

**Comment by bart0sh** [comment]
@liggitt @MHBauer Updated the PR to point to v3.4.10 tagged commit. Please review & merge. Thanks.

File: vendor/go.etcd.io/etcd/etcdserver/etcdserverpb/raft_internal_stringer.go [hunk 1]
```diff
@@ -137,7 +137,7 @@ type loggableValueCompare struct {
 	Result    Compare_CompareResult `protobuf:"varint,1,opt,name=result,proto3,enum=etcdserverpb.Compare_CompareResult"`
 	Target    Compare_CompareTarget `protobuf:"varint,2,opt,name=target,proto3,enum=etcdserverpb.Compare_CompareTarget"`
 	Key       []byte                `protobuf:"bytes,3,opt,name=key,proto3"`
-	ValueSize int                   `protobuf:"bytes,7,opt,name=value_size,proto3"`
+	ValueSize int64                 `protobuf:"varint,7,opt,name=value_size,proto3"`
```
Comment (jingyih) [inline vendor/go.etcd.io/etcd/etcdserver/etcdserverpb/raft_internal_stringer.go:140]: Verified that we are picking up the fix.

**Review by jingyih — COMMENTED** [review]
/lgtm

Thanks!

**Comment by liggitt** [comment]
🎉

**Comment by bart0sh** [comment]
> 🎉

indeed :)

**Pull request merged**